experimenter/experimenter/nimbus_ui/static/js/edit_branches.js

```diff
@@ -12,8 +33,26 @@ const setupCodemirror = (selector, textarea, extraExtensions) => {
     return;
   }
 
+  const highlightStyle = HighlightStyle.define([
+    { tag: tags.bool, color: "#ffaa00ff", themeType: "dark" },
+  ]);
+
   const extensions = [
-    basicSetup,
+    lineNumbers(),
+    drawSelection(),
+    highlightSpecialChars(),
+    foldGutter(),
+    autocompletion(),
+    bracketMatching(),
+    closeBrackets(),
+    highlightSelectionMatches(),
+    indentOnInput(),
+    highlightActiveLine(),
+    syntaxHighlighting(highlightStyle),
+    syntaxHighlighting(defaultHighlightStyle),
+    history(),
+    search(),
+    keymap.of([...defaultKeymap, ...searchKeymap, ...historyKeymap]),
```

Comment:
Yeah this is a bit hairy.  If using dedicated light/dark themes is too complicated let's just go back to your original approach of just overriding the colour of the bools.  Simplest, smallest solution is usually best 🙏